test/test_tags.rb

```diff
@@ -198,6 +214,14 @@ def highlight_block_with_opts(options_string)
           @result
         )
       end
+
+      should "render markdown with pygments with line numbers and custom start line" do
```

Comment:
Note: Support for Pygment will be dropped in 4.0

Reply:
Thanks, that’s good to know. That might factor into API naming as `linenos` comes from Pygment (see below).